openmetadata-docs/content/v1.1.0/deployment/docker/index.md

```diff
@@ -4,22 +4,25 @@ slug: /deployment/docker
 ---
 
 # Docker Deployment
-Deploying OpenMetadata in Docker is a great start! 
+
+This guide will help you setup the OpenMetadata Application using Docker Deployment.
 Before starting with the deployment make sure you follow all the below Prerequisites.
 
 ## Docker Deployment Architecture
 {% image src="/images/v1.1.0/deployment/docker/om_docker_architecture.png" alt="Docker Deployment Architecture" /%}
 
-High-level overview:
+## Prerequisites
 
-- Deploying with MySQL 3306 /PostgreSQL 5432 : Download docker-compose.yml / docker-compose-postgres.yml from the link: https://github.com/open-metadata/OpenMetadata/releases
-/OpenMetadata/releases.
-- We are shipping the Elasticsearch service and Ul at 9200.
-- We are shipping the OpenMetadata server and Ul at 8585.
-- We are shipping the ingestion container (Airflow) at 8080.
-- You can change the port number's according to your requirement.
+### Configure OpenMetadata to use External Database and Search Engine
+
+For Production Deployment using Docker, we recommend to bring your own Databases and ElasticSearch Engine.
```

Comment:
Lets add a partial which will include the versions of dependencies that we can include everywhere